manifests/s/SpanishGovernment/InstaladorDNIe/1.1.0/SpanishGovernment.InstaladorDNIe.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: SpanishGovernment.InstaladorDNIe
+PackageVersion: 1.1.0
+InstallerType: exe
+InstallModes:
+- silent

```

Unfortunately, it appears that the installer switch didn't quite arrive at the PR. 

```suggestion
- silent
InstallerSwitches:
  Silent: /silent
```